### PR TITLE
fix: no pool preds for random heuristic

### PR DESCRIPTION
**Pull request by arthur-thuy**
## Summary:
Issue #274 

### Features:
/

## Checklist:

* [ ] Your code is documented (To validate this, add your module to `tests/documentation_test.py`).
* [x] Your code is tested with unit tests.
* [x] You moved your Issue to the PR state.

### CODE REVIEW

File: baal/active/active_loop.py [hunk 1]
```diff
@@ -79,7 +79,10 @@ def step(self, pool=None) -> bool:
             indices = None
 
         if len(pool) > 0:
-            probs = self.get_probabilities(pool, **self.kwargs)
+            if self.heuristic.__class__.__name__ == "Random":
```
Comment (Dref360) [inline baal/active/active_loop.py:82]: I think we can use `isinstance(self.heuristic, Random)` here.

**Review by Dref360 — CHANGES_REQUESTED** [review]
Thank you for your PR!

Small comment and could we add some test to be sure that it doesn't call `self.get_probabilities`?

File: baal/active/active_loop.py (hunk 1 above)
Comment (arthur-thuy) [reply baal/active/active_loop.py]: Thanks for the comment! I added a unit test that checks whether `self.get_probabilities` is called with `unittest.mock.patch`.

**Review by Dref360 — APPROVED** [review]
Looks great thank you!

LGTM

**Pull request merged**